addons/mail/static/src/scss/discuss.scss

```diff
@@ -327,9 +327,18 @@
         .o_preview_title {
             display: flex;
             .o_preview_name {
-                flex: 0 1 auto;
+                flex: 1 0 auto;
                 @include o-text-overflow;
             }
+            .o_mail_activity_action_buttons {
+                display: flex;
+                flex: 0 1 auto;
```

Comment:
`flex: 1 1 auto;` (take the whole space next to `o_preview_name`)

Reply:
That's not what is requested, it seems icons are aligned on the left.